docs/sql-ref-syntax-qry-select-setops.md

```diff
@@ -19,4 +19,125 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+Set operators are used to combine the results of two queries into a single result. Spark SQL supports three types of set operators:
+- `EXCEPT` and `EXCEPT ALL`
+- `INTERSECT` and `INTERSECT ALL`
+- `UNION` and `UNION ALL`
+
+Note that the queries' result sets must have the same number of columns and compatible data types for the respective columns.
+
+### EXCEPT and EXCEPT ALL
+`EXCEPT` and `EXCEPT ALL` return the rows that are found in one query but not the other query. `EXCEPT` takes only distinct rows while `EXCEPT ALL` does not remove duplicates.
+
+#### Syntax
+{% highlight sql %}
+[ ( ] query [ ) ] EXCEPT [ ALL ] [ ( ] query[ ) ]
+{% endhighlight %}
+
+### INTERSECT and INTERSECT ALL
+`INTERSECT` and `INTERSECT ALL` return the rows that are found in both queries. `INTERSECT` takes only distinct rows while `INTERSECT ALL` does not remove duplicates.
+
+#### Syntax
+{% highlight sql %}
+[ ( ] query [ ) ] INTERSECT [ ALL ] [ ( ] query [ ) ]
+{% endhighlight %}
+
+### UNION and UNION ALL
+`UNION` and `UNION ALL` return the rows that are found in either query. `UNION` takes only distinct rows while `UNION ALL` does not remove duplicates.
+
+#### Syntax
+{% highlight sql %}
+[ ( ] query [ ) ] UNION [ ALL ] [ ( ] query [ ) ]
+{% endhighlight %}
+
+### Examples
+{% highlight sql %}
+-- Use number1 and number2 tables to demonstrate set operators.
+SELECT * FROM number1;
++---+
+|  C|
```

Comment:
super nit though... could the column name be lowercased for distinguishing it from the SQL keywords?